packages/apolloKeyv/src/index.ts

```diff
@@ -0,0 +1,39 @@
+import type {
+  KeyValueCache,
+  KeyValueCacheSetOptions,
+} from "@apollo/utils.keyvaluecache";
+import Keyv from "keyv";
+
+export class ApolloKeyv<V = string> implements KeyValueCache<V> {
+  private readonly keyv: Keyv<V>;
+
+  constructor(keyv?: Keyv<V>) {
+    this.keyv = keyv ?? new Keyv<V>();
+  }
+
+  async get(key: string) {
```

Comment:
Might be nice to show return values on the methods.

Reply:
[d05a548](https://github.com/apollographql/apollo-utils/pull/93/commits/d05a548f5db3b3808d2dad4261e4a322eaffafae)